### PR TITLE
✨Add T&C page to consulting pages, and create a layout for T&C page.

### LINKED ISSUE #285: Consulting - Terms and Conditions
Migrate <https://www.ssw.com.au/ssw/Consulting/Terms-and-Conditions/>

Same URL `consulting/terms-and-conditions`

```[tasklist]
### Tasks
- [ ] Consulting schema - Add a layout picker for consulting pages
- [ ] Existing consulting pages use existing layout
- [x] T&C page uses layout similar to existing website
- [x] Add T&C content
- [x] T&C content needs a CODEOWNER
```

### PR DESCRIPTION
**Pull request by KristenHu**
Closes #285 

Still got many work to do:

- [x] Add agreement form in T&C page
- [x] style agreement form component
- [x] Add CODEOWNER to T&C content

URLs affected: /terms-and-conditions

**Progress as of 29/3/23 by Harry:**
- Has been converted to prose to match previous
Issues with this conversion:
  - Spacing is weird now on /terms-and-conditions
  - Spacing is also weird on homepage (/):

MicrosoftTeams-image
**Figure: No "prose" applied**

image
**Figure: "Prose" classname applied**

Personally, I think it looks better like this - and closer to the v1 site, so I don't really mind this difference. However spacing is still a little weird on both `/ `and `/terms-and-conditions`.

### CODE REVIEW

**Comment by wicksipedia** [comment]
Are you able to make it a little more like <https://www.ssw.com.au/ssw/Consulting/Terms-and-Conditions/>:
- add zebra striping to the table
- t&c clauses in 2 columns

**Review by bradystroud — CHANGES_REQUESTED** [review]
Breadcrumb is missing

image

**Review by bradystroud — CHANGES_REQUESTED** [review]
image

this should be seperate like the v1 
image

**Review by bradystroud — CHANGES_REQUESTED** [review]
Make address box big
image
**Figure: Too small for address**

See v1 page
https://www.ssw.com.au/ssw/Consulting/Terms-and-Conditions/
image

**Review by bradystroud — CHANGES_REQUESTED** [review]
Add a placeholder to this field like the v1

image

File: content/pages/terms-and-conditions.mdx [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    
+    * For partial day bookings all travel time within business hours is billable. Travel time is calculated as the time it takes to make a return trip between SSW office and the respective location. SSW reserves the right to refuse travel to any location for any reason.
+    * If travel is applicable, the Client shall pay for ordinary expenses including accommodation, airfare, and other transport. We rather arrive on time so please book airfares with a decent airline that won't restrict baggage or flexibility options. Accommodation shall be charged at market rates but at no less than AUD$200 + GST per person per night if not booked and paid for by the client. A per diem of AUD$60 + GST per person per day is chargeable for meals if reasonable provision is not made by the Client. If flights or other expenses are purchased by SSW, a 20% admin fee will be charged.
+    
+    Note: The Sydney Metropolitan area is defined by the council areas in the Sydney Inner (SI) and Sydney Outer (SO) regions listed by the [Department of Local Government](http://www.cityofsydney.nsw.gov.au/learn/research-and-statistics/the-city-at-a-glance/metropolitan-sydney).
```
Comment (wicksipedia) [inline content/pages/terms-and-conditions.mdx]: This link results in a 404 - @UlyssesMaclaren do you have a better link?

**Review by wicksipedia — CHANGES_REQUESTED** [review]

File: content/pages/terms-and-conditions.mdx (hunk 1 above)
Comment (UlyssesMaclaren) [reply content/pages/terms-and-conditions.mdx]: someone called me the other day with an alternative that worked. I think it was Harry

**Review by wicksipedia — APPROVED** [review]

**Pull request merged**